hawkular-accounts-feature-pack-resources/src/main/resources/subsystem-templates/hawkular-accounts-ejb3.xml

```diff
@@ -0,0 +1,91 @@
+        <bean-instance-pool-ref pool-name="slsb-strict-max-pool" />
+      </stateless>
+      <singleton default-access-timeout="5000" />
+    </session-bean>
+       <?MDB?>
+    <pools>
+      <bean-instance-pools>
+        <!-- Automatically configure pools. Alternatively, max-pool-size can be set to a specific value -->
+        <strict-max-pool name="slsb-strict-max-pool" derive-size="from-worker-pools" instance-acquisition-timeout="5" instance-acquisition-timeout-unit="MINUTES" />
+        <strict-max-pool name="mdb-strict-max-pool" derive-size="from-cpu-count" instance-acquisition-timeout="5" instance-acquisition-timeout-unit="MINUTES" />
+      </bean-instance-pools>
+    </pools>
+    <caches>
+      <cache name="simple" />
+      <cache name="distributable" aliases="passivating clustered" passivation-store-ref="infinispan" />
+    </caches>
+    <passivation-stores>
+      <passivation-store name="infinispan" cache-container="ejb" max-size="10000" />
+    </passivation-stores>
+    <async thread-pool-name="default" />
+    <timer-service thread-pool-name="default" default-data-store="default-file-store">
+      <data-stores>
+        <file-data-store name="default-file-store" path="timer-service-data" relative-to="jboss.server.data.dir" />
+      </data-stores>
+    </timer-service>
+    <remote connector-ref="http-remoting-connector" thread-pool-name="default" />
+    <thread-pools>
+      <thread-pool name="default">
+        <max-threads count="10" />
+        <keepalive-time time="100" unit="milliseconds" />
+      </thread-pool>
+    </thread-pools>
+       <?IIOP?>
+    <default-security-domain value="other" />
+    <default-missing-method-permissions-deny-access value="true" />
+    <log-system-exceptions value="true" />
+  </subsystem>
+  <supplement name="default">
+    <replacement placeholder="STATEFUL-BEAN">
+      <stateful default-access-timeout="5000" cache-ref="simple" passivation-disabled-cache-ref="simple" />
+    </replacement>
+    <replacement placeholder="MDB">
+      <mdb>
+        <resource-adapter-ref resource-adapter-name="${ejb.resource-adapter-name:activemq-ra.rar}" />
+        <bean-instance-pool-ref pool-name="mdb-strict-max-pool" />
+      </mdb>
+    </replacement>
+  </supplement>
+  <supplement name="full" includes="default">
+    <replacement placeholder="IIOP">
+      <iiop enable-by-default="false" use-qualified-name="false" />
+    </replacement>
+    <replacement placeholder="MDB">
+      <mdb>
+        <resource-adapter-ref resource-adapter-name="${ejb.resource-adapter-name:activemq-ra.rar}" />
+        <bean-instance-pool-ref pool-name="mdb-strict-max-pool" />
+      </mdb>
+    </replacement>
+  </supplement>
+  <supplement name="ha">
+    <replacement placeholder="STATEFUL-BEAN">
+      <stateful default-access-timeout="5000" cache-ref="distributable" passivation-disabled-cache-ref="simple" />
+    </replacement>
+  </supplement>
+  <supplement name="full-ha" includes="full ha" />
```

Comment:
Yes, good catch, I cared solely for the plain standalone.xml to see it working. Everything else incl. ha, domain, etc. needs to be done (if we need it).